globals/card_definitions.gd

```diff
@@ -401,7 +401,7 @@ func get_effect_type_text(effect, card_name_source : String = "", char_effect_pa
 				if char_effect_panel:
 					effect_str += get_effect_text(effect['added_effect'], false, false, false, card_name_source, false)
 				else:
-					effect_str += "Add effect:\n" + get_effect_text(effect['added_effect'], false, false, false, card_name_source, false)
+					effect_str += "Add effect -- " + get_effect_text(effect['added_effect'], false, false, false, card_name_source, false)
```

Comment:
why double dash and not just : or -?
I assume you removed the \n because whatever this was printed in (the choice box?) looks better?

Reply:
Yeah changed for the choice box, and things like "Add Effect: Hit: Do whatever" looked awkward; I can try single dash

Reply:
i'll let you decide if single or double looks better. We just don't currently use that anywhere in the UI, which maybe is what you're looking for.